pages/mesosphere/dcos/services/kubernetes/2.7.0-1.18.6/release-notes/index.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+title: Release Notes
+menuWeight: 10
+excerpt: Release notes for DC/OS Kubernetes version 2.7.0-1.18.6
+---
+
+<!-- This source repo for this topic is https://github.com/mesosphere/dcos-kubernetes-cluster -->
+
+# Version 2.7.0-1.18.6
```

Comment:
Must have accurate release date before we merge.

Reply:
Talked to Jimmi and Simao about the release-date issue. 

There is a bit of a problem with that as the official release date is determined by the approval of the CNCF. In other words, until the CNCF had approved MKE’s new release candidate, we don’t have such date.

We should consider adding the date to the version (if really needed) as a final step --- after the docs itself got approved. Can we do this in another step for now?